### PR TITLE
#69: Add document node factory that wraps java objects

### LINKED ISSUE #69: Add document node factory that wraps java objects
Add a factory for document nodes that wraps java objects

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #69

### CODE REVIEW

File: doc/changes/changes_4.0.0.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Code name:
 
+## Features:
+
+* #69: Add document node factory that wraps java objects
```
Comment (morazow) [inline doc/changes/changes_4.0.0.md:7]:
```suggestion
* #69: Added document node factory that wraps java objects
```

File: src/main/java/com/exasol/adapter/document/documentnode/holder/DoubleHolderNode.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.exasol.adapter.document.documentnode.holder;
+
+import com.exasol.adapter.document.documentnode.DocumentFloatingPointValue;
+
+/**
+ * Implementation of {@link DocumentFloatingPointValue} that simply holds the double value value in a variable.
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/document/documentnode/holder/DoubleHolderNode.java:6]:
```suggestion
 * Implementation of {@link DocumentFloatingPointValue} that simply holds the double value in a variable.
```

File: src/test/java/com/exasol/adapter/document/UdfCallBuilderTest.java [hunk 3]
```diff
@@ -33,7 +26,7 @@ void testBuildForEmptyPlan() throws IOException {
         final RemoteTableQuery remoteTableQuery = getRemoteTableQueryWithOneColumns();
         final QueryPlan queryPlan = new EmptyQueryPlan();
         final String udfCallSql = UDF_CALL_BUILDER.getUdfCallSql(queryPlan, remoteTableQuery);
-        assertThat(udfCallSql, equalTo("SELECT * FROM (VALUES (CAST(NULL AS VARCHAR(123)))) WHERE FALSE"));
+        assertThat(udfCallSql, equalTo("SELECT * FROM (VALUES (CAST(NULL AS  VARCHAR(123)))) WHERE FALSE"));
```
Comment (morazow) [inline src/test/java/com/exasol/adapter/document/UdfCallBuilderTest.java:29]: Is this needed?

**Review by morazow — APPROVED** [review]
Added minor suggestions.

File: src/test/java/com/exasol/adapter/document/UdfCallBuilderTest.java (hunk 3 above)
Comment (jakobbraun) [reply src/test/java/com/exasol/adapter/document/UdfCallBuilderTest.java:29]: Well, actually not. But sql-statement-builder by accident adds two spaces and I did not want to investigate.

**Pull request merged**